### DISCUSSION TITLE
Estimating minimum ensemble size for param perturbation experiment

### DISCUSSION THREAD

**Discussion by adamhb** [NGEET/fates#1054] [Q&A] [answered]
Hi all,

I'm trying to estimate the computational cost of a possible parameter perturbation experiment using Latin hypercube sampling. Does anyone has thoughts on minimum ensemble size per number of pft-level parameters? I see that prior studies have had the following parameter to ensemble size ratio:

Li et al. 2023:  11 : 1500
Buotte et al., 2012: 8 : 360 (by 2 pfts)
Koven et al., 2020: 12 : 576

I'm wondering if those who have done this before wished they had a higher sampling density or if it was sufficient? Given the latin hypercube method of sampling sparsely, could I expect the same sampling density as Li et al., 2023 by just doubling the params and ensemble size (22 parameters and and 3000 ensemble members)?

Any thoughts would be greatly appreciated!
Adam

**Comment by rosiealice** [comment, marked as answer]
Hi Adam. I don't think there is a magic answer here, but one thing you need to try and avoid with LHC is choosing a range where one low value of a parameter can e.g. kill everything in the simulations and then make half of your ensemble pointless. One way of doing that is to do some lower dimensional and resolution tests with parameters we expect might expect to have a strong effect first to eliminate problematics parts of the prior. 

Mat Williams and Anthony Bloom had a nice way of doing this by killing runs that didn't pass through some kind of sanity check after a short period of time. Seems like nowadays that should be easier to automate.

What is the geographical scope of your simulations? I have been doing a lot of mini global ensembles recently that might be useful for getting to the right ballpark.

**Reply by adamhb** [reply]
Thank you @rosiealice for raising this! I can see how that would really reduce the efficiency of finding suitable parameterizations. I'm running on the western slope of Calfornia's Sierra Nevada in the mixed conifer forest zone. Roughly 1000-2000 m in elevation. I'm running with 5 pfts (3 conifers, 1 oak, and 1 shrub) and two fire regimes (pre-colonial frequent low intensity fire regime and a fire suppression regime). I'm considering an ensemble where I perturb pft-level parameters and fire parameters to see if I can get the right fire-vegetation interactions working between these pfts and fire regimes... This could turn out to be too ambitious, but that's the dream for now at least!
